packages/game-bridge/src/index.ts

```diff
@@ -78,13 +83,29 @@ declare function blu_event(event: string, data: string): void;
 // eslint-disable-next-line @typescript-eslint/naming-convention
 declare function UnityPostMessage(message: string): void;
 
-const callbackToGame = (data: object) => {
+interface CallbackToGameData {
+  responseFor: string;
+  requestId: string;
+  success: boolean;
+  error?: string;
+  errorType?: string | null;
+  code?: string;
```

Comment:
`code`, `deviceCode`, `url` and `interval` are only returned for initDeviceFlow. Could they not be part of this CallbackToGameData? Same for `accounts`

Reply:
Added a union type instead so it's clearer